test/harry/main/org/apache/cassandra/harry/model/PartitionState.java

```diff
@@ -410,6 +410,12 @@ public String toString(ValueGenerators valueGenerators)
                        ", lts(" + StringUtils.toString(lts) + ")";
             }
         }
+
+        @Override
+        public String toString()
+        {
+            return toString(partitionState.valueGenerators);
```

Comment:
not related to this patch, but been in the other branch for ages...

when harry models detect a fault and show the rows, it was using the wrong toString which means that the descriptors won't match making it hard to see what happened...